src/WPF/ArcGISRuntime.WPF.Viewer/Samples/MapView/ChooseCameraController/ChooseCameraController.xaml.cs

```diff
@@ -0,0 +1,133 @@
+
+        // Geo element camera controller.
+        private OrbitGeoElementCameraController _orbitPlaneCameraController;
+
+        // Location camera controller.
+        private OrbitLocationCameraController _orbitCraterCameraController;
+
+        public ChooseCameraController()
+        {
+            InitializeComponent();
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create a scene.
+            Scene myScene = new Scene(Basemap.CreateImageryWithLabels());
+
+            // Create a surface for elevation data.
+            Surface surface = new Surface();
+            surface.ElevationSources.Add(new ArcGISTiledElevationSource(_elevationUri));
+
+            // Add the surface to the scene.
+            myScene.BaseSurface = surface;
+
+            // Create a graphics overlay for the scene.
+            GraphicsOverlay sceneGraphicsOverlay = new GraphicsOverlay()
+            {
+                SceneProperties = new LayerSceneProperties(SurfacePlacement.Absolute)
+            };
+            MySceneView.GraphicsOverlays.Add(sceneGraphicsOverlay);
+
+            // Location at the crater.
+            MapPoint craterLocation = new MapPoint(-109.929589, 38.437304, 1700, SpatialReferences.Wgs84);
+
+            // Create the plane symbol and make it 10x larger (to be the right size relative to the scene).
+            ModelSceneSymbol planeSymbol;
+            try
+            {
+                planeSymbol = await ModelSceneSymbol.CreateAsync(_modelUri, 10.0);
+            }
+            catch (Exception ex)
+            {
+                System.Diagnostics.Debug.WriteLine(ex.Message);
+                MessageBox.Show("Loading plane model failed. Sample failed to initalize.");
+                return;
+            }
+
+            // Create a graphic using the plane symbol.
+            Graphic planeGraphic = new Graphic(new MapPoint(craterLocation.X, craterLocation.Y, 5000.0, SpatialReferences.Wgs84), planeSymbol);
+            sceneGraphicsOverlay.Graphics.Add(planeGraphic);
+
+            // Instantiate a new camera controller which orbits a geo element.
+            _orbitPlaneCameraController = new OrbitGeoElementCameraController(planeGraphic, 300.0)
+            {
+                CameraPitchOffset = 30,
+                CameraHeadingOffset = 150
+            };
+
+            // Instantiate a new camera controller which orbits a location.
+            _orbitCraterCameraController = new OrbitLocationCameraController(craterLocation, 6000.0)
+            {
+                CameraPitchOffset = 3,
+                CameraHeadingOffset = 150
+            };
+
+            // Set the starting camera controller.
+            MySceneView.CameraController = _orbitPlaneCameraController;
+
+            // Add events for the radio buttons.
+            OrbitPlaneButton.Checked += Setting_Checked;
```

Comment:
I recommend specifying the event handlers in XAML, leaving the controls disabled until they are ready to be used, then enabling the controls. 

That way, the system handles unsubscribing from the events automatically.